src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -416,6 +430,8 @@ h2o4gpu.kmeans <- function(
   h2o4gpu_model(model, NULL, "KMeans Clustering")
 }
 
+#' @description Principal Components Analysis (PCA)
```

Comment:
Correct name is "Principal Component Analysis" (no "s").  Can we change this?  (in Python if necessary)